.editorconfig

```diff
@@ -16,3 +16,11 @@ indent_size = 4
 # JS should use tabs, to reduce conflicts
 [*.js]
 indent_style = tab
+
+# HTML should use tabs, to reduce conflicts
+[*.html]
+indent_style = tab
+
+# SASS should use tabs, to reduce conflicts
```

Comment:
The comment says tabs, the code says space…

Reply:
Mistake then in one commit ... changed that extra to not change the whole file :(

Reply:
```
[*.scss]
indent_style = space
```

Copy paste mistake ;)